email_recovery/lib/email_recovery/converter/status_converter.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:email_recovery/email_recovery/status.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+class StatusConverter implements JsonConverter<Status, String> {
```

Comment:
If `state=null` then `StatusConverter` does not work correctly. Should be support nullable